client/src/includes/stimulus.stories.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Example Stimulus usage within Storybook, including StimulusWrapper export
+ * Note: Must use .js not .tsx for now
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Controller } from '@hotwired/stimulus';
+import { initStimulus } from './stimulus';
+
+/**
+ * Wrapper around the Stimulus application to ensure that the application
+ * is scoped to only the specific story instance's DOM and also ensure
+ * that the hot-reloader / page switches to not re-instate new applications
+ * each time.
+ *
+ * @example
+ * import Stories from '../includes/stimulus.stories';
+ * const Template = () => <Stories.StimulusWrapper controllers={{ 'w-auto-field': AutoFieldController }}><form data-controller="w-auto-field" /></Stories.StimulusWrapper>
+ */
+class StimulusWrapper extends React.Component {
+  constructor(props) {
+    super(props);
+    this.ref = React.createRef();
+  }
+
+  componentDidMount() {
+    const { controllers = [], debug = false } = this.props;
+
+    // convert convenience object into identifier / controllerConstructor array if needed
+    const definitions = Array.isArray(controllers)
+      ? controllers
+      : Object.entries(controllers).map(
+          ([identifier, controllerConstructor]) => ({
+            identifier,
+            controllerConstructor,
+          }),
+        );
```

Comment:
haha yeah - I actually wrote it that way initially but thought it would be too verbose for those working with this code and opted for the simpler syntax.

I will fix this up tomorrow.